rclcpp/src/rclcpp/node.cpp

```diff
@@ -211,7 +211,8 @@ Node::get_logger() const
 }
 
 rclcpp::CallbackGroup::SharedPtr
-Node::create_callback_group(rclcpp::CallbackGroupType group_type)
+Node::create_callback_group(
```

Comment:
This line break is not required. It was introduced in the demo by the second argument RealTimeClass which is not used here.